review/main.scss

```diff
@@ -1,409 +1,409 @@
-                    margin-bottom: 1rem;
-                }
-
-                &--disabled {
-                    opacity: 0.3;
-                }
-            }
-        }
-
-        &__indexes {
-            top: 50%;
-            right: 4rem;
-            gap: 0.5rem;
-            display: flex;
-            z-index: 999999;
-            position: absolute;
-            flex-direction: column;
-            transform: translateY(-50%);
-
-            #{$className}__index {
-                width: 8px;
-                height: 8px;
-                opacity: 0.5;
-                cursor: pointer;
-                border-radius: 999999px;
-                background-color: #ffffff;
-                border: 1px solid #383938;
-
-                &--active {
-                    opacity: 1;
-                }
-            }
-        }
-    }
-
-    @keyframes comesBottom {
-        0% {
-            opacity: 0;
-            transform: translateY(100%);
-        }
-        100% {
-            opacity: 1;
-            transform: translateY(0%);
-        }
-    }
-
-    @keyframes comesTop {
-        0% {
-            opacity: 0;
-            transform: translateY(-100%);
-        }
-        100% {
-            opacity: 1;
-            transform: translateY(0%);
-        }
-    }
-
-    @keyframes leavesLeft {
-        0% {
-            opacity: 1;
-            transform: translateX(0%);
-        }
-        100% {
-            opacity: 0;
-            transform: translateX(-100%);
-        }
-    }
-
-    @keyframes leavesRight {
-        0% {
-            opacity: 1;
-            transform: translateX(0%);
-        }
-        100% {
-            opacity: 0;
-            transform: translateX(100%);
-        }
-    }
-}
-
-main {
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100vh;
-    position: fixed;
-
-    > .container {
-        top: 0;
-        left: 0;
-        width: 100%;
-        display: flex;
-        min-height: 2500px;
-        position: absolute;
-        flex-direction: column;
-        will-change: transform;
-        justify-content: center;
-
-        > section {
-            width: 100%;
-            height: 100vh;
-            display: flex;
-            color: #383938;
-            position: relative;
-            align-items: center;
-            justify-content: center;
-            background-color: #d2d9f3;
-
-            &:nth-child(odd) {
-                background-color: #f5f6f9;
-            }
-        }
-    }
-}
+*,
+*::before,
+*::after {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    font-size: initial;
+    box-sizing: border-box;
+}
+
+html,
+body {
+    width: 100%;
+    height: 100%;
+    font-size: 16px;
+    line-height: 1.5rem;
+    scrollbar-width: none;
+    font-family: "Inter", sans-serif;
+}
+
+.header {
+    --background-color: #ffffff;
+    --class: ".header";
+
+    top: 0;
+    left: 0;
+    width: 100%;
+    display: flex;
+    position: fixed;
+    z-index: 999999;
```

Comment:
questo z-index non va bene, in generale uno z-index non dovrebbe mai superare 100 o comunque bisogna adottare una scala precisa per evitare poi problemi con altri elementi